src/layers/MultiscaleImageLayer/MultiscaleImageLayerBase.js

```diff
@@ -21,11 +20,7 @@ const defaultProps = {
   lensBorderColor: { type: 'array', value: [255, 255, 255], compare: true },
   lensBorderRadius: { type: 'number', value: 0.02, compare: true },
   transparentColor: { type: 'array', value: null, compare: true },
-  interpolation: {
-    type: 'string',
-    value: INTERPOLATION_MODES.NEAREST,
-    compare: true
-  }
+  interpolation: { type: 'string', value: null, compare: true }
```

Comment:
Right, why not just set this default to `LINEAR` rather than passing down `null` and later setting a default if `null`.